prereqs_linux.sh

```diff
@@ -39,6 +39,11 @@ fi
 
 # Downloading the Google DP library
 git submodule update --init --recursive
-# Removing the Java part 
-rm -rf third_party/differential-privacy/java 
-rm -rf third_party/differential-privacy/examples/java
\ No newline at end of file
+
+# checkout out to particular commit
+cd third_party/differential-privacy && git checkout b7f4c39d9f73d67b34cdbd1b8483e5f72072fc73 && \
+rm -rf third_party/differential-privacy/java third_party/differential-privacy/examples/java
```

Comment:
This path in rm is relative to old location. After `cd third_party/differential-privacy` this rm command should be:
```
rm -rf java examples/java
```